src/main/java/TransactionApp.java

```diff
@@ -0,0 +1,118 @@
+                case DELETE_TRANSACTION:
+                    deleteTransaction();
+                    break;
+                case DISPLAY_INCOMES:
+                    displaySumAmount(TransactionType.INCOME);
+                    break;
+                case DISPLAY_EXPENSES:
+                    displaySumAmount(TransactionType.EXPENSE);
+                    break;
+                case EXIT:
+                    DAO.close();
+                    return;
+                default:
+                    System.out.println("Podałeś nieprawidłową opcję");
+            }
+        }
+    }
+
+    private void modifyTransaction() {
+        System.out.println("Podaj id transakcji, którą chcesz zmodyfikować");
+        Integer id = scanner.nextInt();
+        scanner.nextLine();
+        Transaction transaction = transactionCreator();
+        transaction.setId(id);
+        DAO.update(transaction);
+    }
+
+
+    private void deleteTransaction() {
+        System.out.println("Podaj id transakcji");
+        Integer id = scanner.nextInt();
+        scanner.nextLine();
+        DAO.delete(id);
+
+    }
+
+    private void addNewTransaction() {
+
+        Transaction transaction = DAO.save(transactionCreator());
+        System.out.printf("Obiekt został zapisany z id %s \n", transaction.getId());
+
+    }
+
+    private void displaySumAmount(TransactionType transactionType) {
+        Double byTransactionType = DAO.findByTransactionType(transactionType);
+
+        System.out.println("Suma to: " + byTransactionType);
+    }
+
+
+    private Transaction transactionCreator() {
+        boolean doTheLoop = true;
+        TransactionType type = null;
+
+        while (doTheLoop) {
+            System.out.println("Podaj typ transakcji");
+            System.out.println("1. Przychód");
+            System.out.println("2. Wydatek");
+            String option = scanner.nextLine();
+            if (option.equals("1")) {
+                type = TransactionType.INCOME;
+                doTheLoop = false;
+            } else if (option.equals("2")) {
+                type = TransactionType.EXPENSE;
+                doTheLoop = false;
+            } else {
+                System.out.println("Podana wartość jest nieprawidłowa. Spróbuj ponownie");
+            }
+        }
+        System.out.println("Podaj opis transakcji");
+        String description = scanner.nextLine();
+        System.out.println("Podaj kwotę transakcji");
+        double amount = scanner.nextDouble();
+        scanner.nextLine();
+        return new Transaction(type, description, amount, LocalDate.now());
+    }
+
+    private void printMenu() {
+        System.out.println("1. Dodaj transakcję");
+        System.out.println("2. Zmodyfikuj transakcję");
+        System.out.println("3. Usuń transakcję");
+        System.out.println("4. Wyświetl wszystkie przychody");
+        System.out.println("5. Wyświetl wszystkie wydatki");
+        System.out.println("0. Zakończ program");
```

Comment:
Po wprowadzeniu stałych powinieneś ich używać wszędzie - także tutaj. Jeżeli coś się zmieni, to nie trzeba wprowadzać poprawek w kilku miejscach